site/en/install/gpu.md

```diff
@@ -93,75 +93,52 @@ This section shows how to install CUDA® 11 (TensorFlow >= 2.4.0) on Ubuntu
 Caution: [Secure Boot](https://wiki.ubuntu.com/UEFI/SecureBoot){:.external}
 complicates installation of the NVIDIA driver and is beyond the scope of these instructions.
 
-
-#### Ubuntu 18.04 (CUDA 11.0)
+#### Ubuntu 18.04 (CUDA 11.2)
 
 <pre class="prettyprint lang-bsh">
 # Add NVIDIA package repositories
-<code class="devsite-terminal">wget https://developer.download.nvidia.com/compute/cuda/repos/ubuntu1804/x86_64/cuda-ubuntu1804.pin</code>
-<code class="devsite-terminal">sudo mv cuda-ubuntu1804.pin /etc/apt/preferences.d/cuda-repository-pin-600</code>
-<code class="devsite-terminal">sudo apt-key adv --fetch-keys https://developer.download.nvidia.com/compute/cuda/repos/ubuntu1804/x86_64/7fa2af80.pub</code>
-<code class="devsite-terminal">sudo add-apt-repository "deb https://developer.download.nvidia.com/compute/cuda/repos/ubuntu1804/x86_64/ /"</code>
-<code class="devsite-terminal">sudo apt-get update</code>
-
-<code class="devsite-terminal">wget http://developer.download.nvidia.com/compute/machine-learning/repos/ubuntu1804/x86_64/nvidia-machine-learning-repo-ubuntu1804_1.0.0-1_amd64.deb</code>
-
-<code class="devsite-terminal">sudo apt install ./nvidia-machine-learning-repo-ubuntu1804_1.0.0-1_amd64.deb</code>
-<code class="devsite-terminal">sudo apt-get update</code>
-
-<code class="devsite-terminal">wget https://developer.download.nvidia.com/compute/machine-learning/repos/ubuntu1804/x86_64/libnvinfer7_7.1.3-1+cuda11.0_amd64.deb</code>
-<code class="devsite-terminal">sudo apt install ./libnvinfer7_7.1.3-1+cuda11.0_amd64.deb</code>
+# Note: For Ubuntu version other than 18.04 or CPU arch other than x86,
+# replace 'ubuntu1804' and/or 'x86_64' as needed in the following URL.
+<code class="devsite-terminal">wget https://developer.download.nvidia.com/compute/cuda/repos/ubuntu1804/x86_64/cuda-keyring_1.0-1_all.deb</code>
+<code class="devsite-terminal">sudo dpkg -i cuda-keyring_1.0-1_all.deb</code>
 <code class="devsite-terminal">sudo apt-get update</code>
 
 # Install development and runtime libraries (~4GB)
 <code class="devsite-terminal">sudo apt-get install --no-install-recommends \
-    cuda-11-0 \
-    libcudnn8=8.0.4.30-1+cuda11.0  \
-    libcudnn8-dev=8.0.4.30-1+cuda11.0
+    cuda-11-2 \
+    libcudnn8=8.1.0.77-1+cuda11.2  \
+    libcudnn8-dev=8.1.0.77-1+cuda11.2 \
+    libnvinfer8=8.2.4-1+cuda11.4 \
+    libnvinfer-dev=8.2.4-1+cuda11.4 \
+    libnvinfer-plugin8=8.2.4-1+cuda11.4 \
+    libnvinfer-plugin-dev=8.2.4-1+cuda11.4
 </code>
-# Reboot. Check that GPUs are visible using the command: nvidia-smi
 
-# Install TensorRT. Requires that libcudnn8 is installed above.
```

Comment:
Did you mean to remove the TensorRT installation instructions? Have they changed?

The PR description only notes that TensorRT is available elsewhere, I assume it still needs to be installed?

If this section does need to be removed, then the "software requirements" section will need to be updated to remove the "install tensorrt" entry.

Reply:
Thanks @mihaimaruseac Can you also please check the TensorRT section ?

Reply:
Thanks for that info - please update the "software requirements" section to remove the "install tensorrt" entry.

Reply:
Instead I added a comment noting that libnvinfer optional and needed for TensorRT.
PTAL.